controllers/static_controller.php

```diff
@@ -9,35 +9,6 @@ public function landing() {
     public function error() {
         require_once('views/static/error.php');
     }
-
-    public function registration() {
-        require_once('views/static/registration.php');
-    }
-
-    public function searchEvents() {
-        require_once('views/static/search_events.php');
-    }
-
-    public function login() {
-        require_once('views/static/login.php');
-    }
-
-    public function profile() {
-        require_once('views/static/profile.php');
-    }
-
-    public function logout() {
-        require_once('views/static/logout.php');
-    }
-
-    public function terms(){
-        require_once('views/static/terms.php');
-    }
-
-    public function contact() {
```

Comment:
If you remove the contact here, please don't forget to add it in the dynamic controller. This new page got merged yesterday.